autobahn/src/main/java/io/crossbar/autobahn/wamp/messages/Event.java

```diff
@@ -62,8 +71,14 @@ public static Event parse(List<Object> wmsg) {
         marshaled.add(MESSAGE_TYPE);
         marshaled.add(subscription);
         marshaled.add(publication);
-        // Empty details.
-        marshaled.add(Collections.emptyMap());
+        Map<String, Object> details = new HashMap<>();
+        if (topic != null) {
+			details.put("topic", topic);
+		}
```

Comment:
Could you fix the indentation here. Also note that tabs are 4 spaces in this project so if you could change that in a couple other places that would be cool :)